### ISSUE TITLE
Using domain name in ROOT_URL cause websocket issue

### ISSUE THREAD

**Issue by minyixiecc** [meteorhacks/cluster#119]
I have an issue on using the cluster which cause websocket handshake failing. 

I set the ROOT_URL = "http://domain-name" (example, http://www.xyz.com.au), and set endpoint in a options object in code before cluster.register:

`var options = { endpoint: "ip-address:port"}`

For example, endpoit:"192.168.2.1:80".

After I run meteor, the websocket handshake fails. If I use ip address to ROOT_URL, everything works fine. 

I also tries ROOT_URL = "http://domain-name:port" but no luck.

If you have any clue, please let me know. Thank you for your help.

**Comment by MasterJames** [comment]
Endpoint has an N missing you have endpoit maybe that is the issue?

**Comment by minyixiecc** [comment]
@MasterJames Thank you but that is "point" in my code.

**Comment by MasterJames** [comment]
Ah good. So can you access the port from the target machine? 
You can use SSH or even docker exec with bash or even curl directly to see if it's blocked by firewall rules or something?

**Comment by minyixiecc** [comment]
Yes, i can access it using SSH. My server is on aws, ec2 instance. I set
the port, 80 to be open to all traffic.
